docs/src/main/asciidoc/kafka/kafka_overview.adoc

```diff
@@ -151,12 +151,12 @@ Flag to set the binder health as `down`, when any partitions on the topic, regar
 Default: `false`.
 
 spring.cloud.stream.kafka.binder.certificateStoreDirectory::
-When the truststore or keystore certificate location is given as a classpath URL (`classpath:...`), the binder copies the resource from the classpath location inside the JAR file to a location on the filesystem.
+When the truststore or keystore certificate location is given as a non-local file system resources, the binder copies the resource from the path (which is convertible to org.springframework.core.io.Resource) to a location on the filesystem.
```

Comment:
Could you add the various formats that it supports now? (`CLASSPATH`, `HTTP` etc. as examples)